src/schemes/boundary/dummy_particles/dummy_particles.jl

```diff
@@ -151,6 +151,16 @@ function create_cache_model(viscosity::Nothing, n_particles, n_dims)
     return (;)
 end
 
+function create_cache_model(viscosity::ArtificialViscosityMonaghan, n_particles, n_dims)
+    ELTYPE = eltype(viscosity.alpha)
+
+    wall_velocity = zeros(ELTYPE, n_dims, n_particles)
+
+    @warn "`ArtificialViscosityMonaghan` for `BoundaryModelDummyParticles` has not been verified yet."
+
```

Comment:
Do we want to have a warning here? I'd say it's enough to have a warning in the docs.